README.md

```diff
@@ -361,12 +361,7 @@ after making sure the proper environment variables are set (He_Sub, He_App_RG, I
 
 ```bash
 
-### TODO - this won't work since the instructions clone the language repo, not the helium repo
-### One option is to make dashboard setup a separate md file and include instructions for cloning the helium repo
-### Another option is to add the dashboard files to each of the language repos
-### Another option is to curl the files from helium <- I like this option best
-
-cd docs/dashboard
+curl -s https://raw.githubusercontent.com/retaildevcrews/helium/master/docs/dashboard/Helium_Dashboard.json > Helium_Dashboard.json
```

Comment:
why are the sed commands using &&

I can't figure out a reason why

Would it be a better developer experience to create a small bash script instead?

Reply:
The bash script will certainly help, but I think it'll be a little more of a hassle for the user to copy/paste and run directly as opposed to the normal bash commands.